index.d.ts

```diff
@@ -0,0 +1,50 @@
+type TimeWindow = number; // e.g. 60 * 1000
+
+interface Snapshot {
```

Comment:
Sorry, I missed this in the first comments (I forgot the context...). This `Snapshot` is more like `Histogram`. The `getSnapshot()` method aggregates multiple histograms in the rolling window and returns a snapshot histogram.